### PR TITLE
GUI: add CTRL-TAB/CTRL-SHIFT-TAB keyboard shortcuts to switch tabs

### PR DESCRIPTION
**Pull request by agordon**

### CODE REVIEW

File: src/tabmanager.cc [hunk 1]
```diff
@@ -101,6 +101,44 @@ void TabManager::closeTabRequested(int x)
...
+
+void TabManager::nextTab()
+{
+    assert(tabWidget != nullptr);
+
+    if (tabWidget->count()<=1)
+        return;
+
+    int idx = (tabWidget->currentIndex()+1) % tabWidget->count();
+    PRINTB("NextTab, new-idx=%d", idx);
```
Comment (t-paul) [inline src/tabmanager.cc]: Please remove the output or convert to debug (`PRINTDB`) so it's not shown without being specifically enabled via `--debug`.

File: src/tabmanager.cc [hunk 2]
```diff
@@ -101,6 +101,44 @@ void TabManager::closeTabRequested(int x)
...
+    assert(tabWidget != nullptr);
+
+    if (tabWidget->count()<=1)
+        return;
+
+    int idx = tabWidget->currentIndex()-1;
+    if (idx<0)
+        idx = tabWidget->count()-1;
+
+    PRINTB("PrevTab, new-idx=%d", idx);
```
Comment (t-paul) [inline src/tabmanager.cc]: Same here.

Comment (agordon) [reply src/tabmanager.cc]: done

**Comment by agordon** [comment]
@t-paul thanks!
(before any merges this, please note it also contains #3055. if you need it separate, let me know)

**Comment by t-paul** [comment]
Yes, I noticed. No problem, it will probably auto-close when merging this PR.

**Pull request merged**